_data/projects/Data Structure and Algorithms.yml

```
@@ -0,0 +1,12 @@
+name: Data Structure and Algorithms
+desc: Repository for implementing all Data Structre and Algorithms
+site: https://github.com/kshitizsaini113/Data-Structure-and-Algorithms
+tags:
+  - c
+  - c++
+  - data-structure
+  - algorithm
+  - contributions
+upforgrabs:
+  name: Data Structure and Algorithms
```

I believe this is what your label should be:

```suggestion
  name: up-for-grabs
```